models/db/summaryParts.js

```diff
@@ -18,7 +18,7 @@ function retrieve(filter) {
 async function retrieveBlock(filter, firstID) {
   // whereBetween will include last number.  If we're at firstID = 1 and want to show 5
   // in a block, 1 + 5 = 6.  Thus, 6 will be included, so -1 to set it back to 1-5
-  const lastID = firstID + (parseInt(process.env.BLOCK_LENGTH, 10) || 5) - 1;
+  const lastID = firstID + (parseInt(process.env.BLOCK_LENGTH, 10) || 3) - 1;
```

Comment:
FYI, we can just change the BLOCK_LENGTH environment variable to adjust this on the fly.  The 5 and now 3 are just default values if the person running this doesn't have a BLOCK_LENGTH defined.